lib/machine-loader.js

```diff
@@ -34,15 +36,40 @@ function getConfig (defaultConfig) {
 }
 
 function getMachineNames (config) {
+  const fullyFunctionalStatus = {label: 'Fully functional', type: 'success'}
```

Comment:
Hmm, thinking more about this we just want fully functional if we're not stuck and the machine is pinging. There's no reason to show the operator "Fully functional" and "Stuck" at the same time.

Reply:
Just noticed that this is from another PR. Fix that there and rebase. This also has conflicts